### ISSUE TITLE
Different size images cause display problems in carousel view

### ISSUE THREAD

**Issue by dmnyc** [damus-io/damus#963]
When there are multiple images with different dimensions in a carousel, the display height changes when swiping. 

Some images get resized when swiping back to the beginning, so they appear off-center.

Example notes:
note1gdcaelddwxckfvgvpgya2kef2zpsdv3mx7akapz0sy7mm0424k8sjxkh57

The second and fourth image in this carousel are not displaying the correct width in relation to the others.

Screen recording:
https://nostr.build/av/nostr.build_a4e267ff6f4d2b4d62f058153d1fd1b9af6f9066790ed4e291437adb2726e3e2.mp4

**Comment by alltheseas** [comment]
@dmnyc is this fixed for you?

**Comment by jb55** [comment]
On Thu, Jun 01, 2023 at 10:10:55AM -0700, alltheseas wrote:
>@dmnyc is this fixed for you?

This should be fixed now thanks to Ben's new carousel updates

**Comment by dmnyc** [comment]
Yes. The new update looks terrific. No images are out of alignment.

I really like the new dot indicators. It would be nice if these could also appear when the images are enlarged.

**Comment by alltheseas** [comment]
> Suhail broke the Photo Gallery [bug] #1170

Great! Can you kindly close this issue 🙏 ?

I added your suggestion of adding the new dot indicators to the enlarged view here: https://github.com/damus-io/damus/issues/1227

**Comment by dmnyc** [comment]
Great, thank you!
